src/vs/base/parts/quickopen/browser/quickOpenWidget.ts

```diff
@@ -335,7 +336,15 @@ export class QuickOpenWidget implements IModelProvider {
 		}
 
 		this.applyStyles();
-
```

Comment:
I prefer to keep the newline here, looks logically too close otherwise imho:

![image](https://user-images.githubusercontent.com/900690/29318819-edb1b726-81d1-11e7-97af-d500eec526a4.png)